product_variant_configurator/models/product_template.py

```diff
@@ -44,12 +44,13 @@ def onchange_no_create_variants(self):
                 }
             }
 
-    @api.model
-    def create(self, vals):
+    @api.model_create_multi
+    def create(self, vals_list):
         if "product_name" in self.env.context:
```

Comment:
Hi @pedrobaeza and @alexbottemanne 

Where is this context set in the logic?

Reply:
AFAIK, it was a mechanism for assuring the proper name when creating the template from the variant itself. Not sure if it has been perverted through versions or not needed anymore.